scripts/modules/MsIndustryLinks/publish/application/AzureMarketplacePublishing.md

```diff
@@ -0,0 +1,98 @@
+# Publishing Azure Application Offer to Azure Marketplace
+
+The **MsIndustryLinks** module contains commands to automate some of the steps required to publish a Logic App-based Industry Link to Azure Marketplace.
+
+Use the following steps to prepare your Azure Marketplace package and create an offer:
+
+## Step 1. Create a Partner Center account
+
+See [Create a Microsoft Partner Center account](https://learn.microsoft.com/en-us/power-platform/developer/appsource/register-microsoft-partner-network) to create a Partner Center account.
+
+## Step 2. Associated an Azure AD application with your Partner Center account
+
+### Onboard to the Partner Center API
+
+See [Partner Center submission API onboarding](https://learn.microsoft.com/en-us/partner-center/marketplace/submission-api-onboard) to associate an Azure AD application with your Partner Center account.
+
+### Login to Azure as the Azure AD application
+
+```
+az login --service-principal -u <CLIENT-ID> -p <CLIENT-SECRET> --tenant <TENANT-ID>
+```
+
+## Step 3. Install Required Tools
```

Comment:
Should step 2 and 3 be swapped? You can't login without the package installed :P

Reply:
Whoops, yes, will swap